src/actions/export.js

```diff
@@ -1,15 +1,15 @@
 'use strict';
 require('log-prefix')(() => `[${new Date().toISOString()}] Doctor-core: %s`);
 const loadAccount = require('../util/loadAccount');
-const {startSpinner, stopSpinner} = require('../util/spinner');
+const { startSpinner, stopSpinner } = require('../util/spinner');
```

Comment:
If possible, move these spinners to the CLI code.

Reply:
Yes, will do this in separate PR when `doctor-cli` will refer to `doctor-core`